src/main/java/uk/gov/ons/census/fwmt/jobservice/service/mapper/SpgMapper.java

```diff
@@ -0,0 +1,95 @@
+package uk.gov.ons.census.fwmt.jobservice.service.mapper;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.ons.census.fwmt.jobservice.dto.rm.FieldworkFollowup;
+import uk.gov.ons.census.fwmt.jobservice.dto.tm.Address;
+import uk.gov.ons.census.fwmt.jobservice.dto.tm.CaseType;
+import uk.gov.ons.census.fwmt.jobservice.dto.tm.Contact;
+import uk.gov.ons.census.fwmt.jobservice.dto.tm.Geography;
+import uk.gov.ons.census.fwmt.jobservice.dto.tm.Location;
+import uk.gov.ons.census.fwmt.jobservice.dto.tm.PutCaseRequest;
+
+import java.util.List;
+
+@Slf4j
+@Service
```

Comment:
make this a final static class with a static method, and take out the @Service. Once we start working on the other message types we can decide how the convertors/mappers work, alternative to the current CometConverters.